### DISCUSSION TITLE
MorphTo Relationship didn't load when the data is too many

### DISCUSSION THREAD

**Discussion by khatabwedaa** [laravel/nova-issues#5864] [Q&A] [answered]
- Laravel Version: 10.x.x
- Nova Version: 4.0
- PHP Version: 8.1
- Database Driver & Version: MYSQL
- Operating System and Version: Ubuntu 22.04
- Browser type and version: Chrome
- Reproduction Repository: Private

### Description: 
I have users, twitter_user tables, and components tables these three tables have a Morph relationship 

Relationship Code 
```php
// Component Model
public function author()
{
    return $this->morphTo()->withDefault(function () {
        return new TwitterUser([
            'name' => 'Anonymous',
            'anon' => true,
            'slug' => 'anon',
            'avatar' => '/svg/anon.svg',
        ]);
    });
}

//------------------------------------------------------------------------

// User Model 
public function components()
{
    return $this->morphMany(Component::class, 'author');
}

//----------------------------------------------------------------------

// Nova Field
MorphTo::make('author')->types([
    User::class,
    TwitterUser::class,
])->nullable(),
```

So the issue is when I select the user option from the menu the app crashes after some debugging I found the reason is the user's table has much data over 14k rows. This issue happened in the create and edit pages How can I paginate the relation on the edit page I need just the author that has created the component if I want to update and then load the users with pagination.

![Screenshot from 2023-09-05 22-53-13](https://github.com/laravel/nova-issues/assets/38974561/147774f0-6480-4be7-9151-005f507fc5f7)

**Comment by crynobone** [comment, marked as answer]
You should use `MorphTo::searchable()` for such usage.

**Reply by khatabwedaa** [reply]
That's work, Thanks @crynobone :smile:
